### PR TITLE
Change slides/.md file

### PR DESCRIPTION
**Pull request by alin-balan**
Caption update

## Done
- branched
- made a commit

## Todo
- [ ] merge branch
- [ ] delete branch

### CODE REVIEW

File: _slides/06-alin-balan.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
```
Comment (tlokken) [inline _slides/06-alin-balan.md:5]: Wow! Great work here.

**Review by tlokken — APPROVED** [review]
Nicely done!

**Comment by alin-balan** [comment]
PR comment

File: _slides/06-alin-balan.md (hunk 1 above)
Comment (alin-balan) [reply _slides/06-alin-balan.md:5]: Resolve conversation

**Pull request merged**